### ISSUE TITLE
Script is missing -k parameter before the server license key

### ISSUE THREAD

**Issue by jon-allenby** [aws-quickstart/quickstart-tableau-server#110]
https://github.com/aws-quickstart/quickstart-tableau-server/blob/6d01ac854ab1ebe0a1bebd0992f6f847f7b2d138/templates/tableau-server-linux.template.yaml#L494

The server license key is missing a preceding -k so when you run this script it incorrectly thinks that the license key is the package file:

```
Package file <license key> not found. Canceling.
Provide the package file (rpm or deb) as last argument.
```

**Comment by vsnyc** [comment]
Thanks @jon-allenby for reporting this . One workaround is to start a trial deployment and then update the license key later using Tableau GUI/CLI. I'm testing a fix in #111.

**Comment by vsnyc** [comment]
@jon-allenby - this should be fixed. Our default pipeline tests without a key, let us know if you find any issues.
